### PR TITLE
Feature  remove acme challenge resources

### PR DESCRIPTION
**Pull request by AngeloB-AIS**
## REMOVED ACME CHALLENGE RESOURCES
The ACME challenge resources are being removed in this PR. It turned out that they weren't actually being used. This was found when I isolated incoming traffic to the K8s cluster to the VPN server but the TLS certificates issued by cert-manager were still being validated (which requires the ACME challenge service to send a request to the cluster). It turns out that cert-manager temporarily stands up its own resources (ingress, pod, etc) to handle the ACME challenges and due that ingress lacking the source whitelist annotation, the ACME challenge service is still able to reach the cert-manager temporary pod and validate the challenge.

This, in effect, also removed resources that are unnecessarily consuming host resources.

### CODE REVIEW

File: charts/multiwoven/values.yaml
Comment (RafaelOAiSquared) [inline charts/multiwoven/values.yaml:1]: What's the purpose of adding the `-community` sufix?

**Review by RafaelOAiSquared — CHANGES_REQUESTED** [review]
According to the PR description, this PR is about removing the ACME challenge resources, but adds a `appEnv` suffix, changing all names and namespaces.

**Comment by AngeloB-AIS** [comment]
@RafaelOAiSquared I created this branch off of the appEnv change branch so I could keep working. This is why I asked if I should merge this branch into that branch before merging it into main first.

File: charts/multiwoven/values.yaml
Comment (AngeloB-AIS) [reply charts/multiwoven/values.yaml:1]: Bc "community" is the default value for appEnv. Without it, the other resources will be unable to find those resources in the cluster.

**Pull request closed without merging**